Runigram.java

```diff
@@ -60,48 +89,98 @@ private static void print(Color c) {
 	// For example, to check that some image processing function works correctly,
 	// we can apply the function and then use this function to print the resulting image.
 	private static void print(Color[][] image) {
-		//// Replace this comment with your code
+		for(int i=0; i<image.length; i++)
+		{
+			for(int j=0; j<image[0].length; j++)
+			{
+				System.out.print("(");
+				System.out.printf("%3s,",image[i][j].getRed());
+				System.out.printf("%3s,",image[i][j].getGreen());
+				System.out.printf("%3s",image[i][j].getBlue());
+				System.out.print(")  ");
+			}
+			System.out.println();
+		}
 	}
 	
 	/**
 	 * Returns an image which is the horizontally flipped version of the given image. 
 	 */
 	public static Color[][] flippedHorizontally(Color[][] image) {
-		//// Replace the following statement with your code
-		return null;
+		Color[][] secondImage=new Color[image.length][image[0].length];
+		for(int i=0; i<secondImage.length; i++)
+		{
+			for(int n=0, j=image[0].length-1 ;n<image[0].length && j>=0 ;j--, n++)
+			{

```

Comment:
Do not use that syntax - loop iteration might involve only one iterating variable so it can be clear.
```suggestion
			for(int j=0 ; j < image[0].length ;++)
			{
				int n = image[0].length-1 - i;

```